### PR TITLE
Action Button Replacement

### PR DESCRIPTION
**Pull request by zakariassoul**
Action Button Replacement

moved the action button so it does not jump around when streaming.

### CODE REVIEW

**Comment by zakariassoul** [comment]
Screenshot 2024-01-09 at 7 06 43 PM

File: python/src/aiconfig/editor/client/src/components/prompt/PromptActionBar.tsx [hunk 1]
```diff
@@ -102,12 +102,12 @@ export default memo(function PromptActionBar({
               )}
             </Tabs>
           </Container>
-          <RunPromptButton
+          {/* <RunPromptButton
```
Comment (saqadri) [inline python/src/aiconfig/editor/client/src/components/prompt/PromptActionBar.tsx:105]: Why commented out?

File: python/src/aiconfig/editor/client/src/components/prompt/PromptActionBar.tsx [hunk 2]
```diff
@@ -116,12 +116,12 @@ export default memo(function PromptActionBar({
               <IconClearAll />
             </ActionIcon>
           </Flex>
-          <RunPromptButton
+          {/* <RunPromptButton
```
Comment (saqadri) [inline python/src/aiconfig/editor/client/src/components/prompt/PromptActionBar.tsx:119]: Why is this commented out?

**Review by saqadri — COMMENTED** [review]
I think it clashes a bit with the json button next to it since they aren't the same size. Will defer to @rholinshead for feedback

**Comment by rholinshead** [comment]
> I think it clashes a bit with the json button next to it since they aren't the same size. Will defer to @rholinshead for feedback

Ya, I think we should keep the JSON editor toggle above for consistency with settings view

**Comment by rholinshead** [comment]
How does this run button look for long inputs?

**Comment by zakariassoul** [comment]
@rholinshead & @saqadri in a previous iteration, i made the JSON toggle in the next row. It's still discoverable, but not really critical to take a front and center place. I tried to push this change with this PR but for the life of me i could not figure out how the template was put together, i spent too much time to justify moving a button so i just pushed as is. 

@rholinshead  take a look at the screenshot below and if all good, can you please make the change in a different diff?

Screenshot 2024-01-09 at 9 52 22 PM

**Comment by zakariassoul** [comment]
@rholinshead  this is how it would look like for long input 
Screenshot 2024-01-09 at 9 55 39 PM

**Comment by rholinshead** [comment]
Thanks @zakariassoul  -- https://github.com/lastmile-ai/aiconfig/pull/857/files is what I was able to get to match those screenshots as best I could

**Pull request closed without merging**